draft-ietf-quic-manageability.md

```diff
@@ -562,14 +574,16 @@ Changes to this behavior have been discussed in the working group, but there
 is no current proposal to implement these changes:  see
 https://github.com/quicwg/base-drafts/issues/602.
 
-## Round-trip time measurement {#sec-rtt}
+## Round-Trip Time (RTT) Measurement {#sec-rtt}
 
 Round-trip time of QUIC flows can be inferred by observation once per flow,
 during the handshake, as in passive TCP measurement; this requires parsing of
 the QUIC packet header and recognition of the handshake, as illustrated in
 {{handshake}}. It can also be inferred during the flow's lifetime, if the
 endpoints use the spin bit facility described below and in
-{{?QUIC-SPIN=I-D.ietf-quic-spin-exp}}.
+{{QUIC-TRANSPORT}}, section X.X.
```

Comment:
```suggestion
{{QUIC-TRANSPORT}}, section 17.3.1.
```
(This is the present location in the editor's draft. Otherwise, we can omit the section reference.)